packages/frontend/src/components/Explorer/ResultsCard/ExplorerResultsNonIdealStates.tsx

```diff
@@ -228,8 +228,10 @@ export const ExploreErrorState = ({
         title="Error loading results"
         description={
             <Fragment>
-                {errorDetail?.message ||
-                    'There was an error loading the results'}
+                <Text style={{ whiteSpace: 'pre-wrap' }}>
```

Comment:
I know it's a bit of a pain, but we should use `createStyles` or CSS modules for styling for the migration to Mantine v8.